tests/lib/prepare.sh

```diff
@@ -994,7 +994,7 @@ EOF
 set default=0
 set timeout=2
 menuentry 'flash-all-snaps' {
-linux $DEVPREFIX/vmlinuz root=$ROOT ro init=$IMAGE_HOME/reflash.sh console=ttyS0
+linux $DEVPREFIX/vmlinuz root=$ROOT ro init=$IMAGE_HOME/reflash.sh console=tty1 console=ttyS0
```

Comment:
Might be wrong, but IIRC in this setup the console will appear on tty1 only, while ttyS0 is what the GCP monitors and presents in the UI. Maybe it's be better if we try something like:

```
    consoleport=ttyS0
    if [ -n "SPREAD_QEMU_GUI=1" ]; then
        # only makes sense when running spread locally
        consoleport=tty1
    fi
   cat >/boot/grub/grub.cfg <<EOF
....
linux $DEVPREFIX/vmlinuz root=$ROOT ro init=$IMAGE_HOME/reflash.sh console=tty1 console=$consoleport
...
EOF
```

Reply:
Or, nvm, I'm reading https://elixir.bootlin.com/linux/latest/source/Documentation/admin-guide/serial-console.rst#L39 and the example is exactly VGA and serial, so I guess having both there is ok.